pytorch_pfn_extras/handler/_logic.py

```diff
@@ -214,22 +215,35 @@ def _forward(self, model: torch.nn.Module, batch: Any) -> Any:
 
     def _backward(self, outputs: Dict[str, Any]) -> None:
         target = _normalize_outputs(outputs)
+        backward_outputs = None
+        if type(self.backward_outputs) is str:
+            backward_outputs = set((self.backward_outputs,))
+        elif self.backward_outputs is not None:
+            backward_outputs = set(self.backward_outputs)
 
         for k, v in target.items():
```

Comment:
How about looping by `for k in self.backward_outputs` when `backward_outputs` is not None?